cnf-certification-test/performance/suite.go

```diff
@@ -79,6 +90,66 @@ var _ = ginkgo.Describe(common.PerformanceTestKey, func() {
 	// Scheduling related tests ends here
 })
 
+func testLimitedUseOfExecProbes(env *provider.TestEnvironment) {
+	var compliantObjects []*testhelper.ReportObject
+	var nonCompliantObjects []*testhelper.ReportObject
+	counter := 0
+	for _, put := range env.Pods {
+		for _, cut := range put.Containers {
+			if cut.LivenessProbe != nil && cut.LivenessProbe.Exec != nil {
+				counter++
+				if cut.LivenessProbe.PeriodSeconds > minExecProbePeriodSeconds {
+					tnf.ClaimFilePrintf("Container %s is using exec probes, PeriodSeconds of LivenessProbe: %s", cut,
+						cut.LivenessProbe.PeriodSeconds)
```

Comment:
Add the actual exec probe period in the report objects: 
compliantObjects = append(compliantObjects, testhelper.NewContainerReportObject(put.Namespace, put.Name, cut.Name, fmt.Sprintf("Liveness exec probe has a PeriodSeconds greater than 10 ( %d seconds)", cut.LivenessProbe.PeriodSeconds), true))